mmconfig

```diff
@@ -2,191 +2,167 @@
+        <hbox space-fill="false">
+          <text wrap="false" width-chars="60" xalign="0">
+            <label>Please choose Y or N to test and adjust volume during transcoding:</label>
+          </text>
+          <vbox>
+            <radiobutton>
+              $(funcrdoCreate VOLADJUSTY "Y")
+            <radiobutton active="true">
+              $(funcrdoCreate VOLADJUSTN "N")
+          </vbox>
+          $(funcEntry DVDLABELPREFIX "Please enter the prefix you would like added to the filename in makedvd. You may leave this blank:" "80")
+        </hbox>
+        <hbox space-expand="true">
+          <text>
+            <label>Please choose Y or N to enable database logging:</label>
+          </text>
+          <vbox>
+            <radiobutton>
+              <action>if true enable:PREMIS_PROFILE</action>
+              <action>if true enable:PREMIS_NAME</action>
+              $(funcrdoCreate PREMIS_DBY "Y")
+            <radiobutton active="true">
+              <action>if true disable:PREMIS_PROFILE</action>
+              <action>if true disable:PREMIS_NAME</action>
+              $(funcrdoCreate PREMIS_DBN "N")
+          </vbox>
+          $(funcEntry PREMIS_PROFILE "Enter the name of the database profile:" "60")
+          $(funcEntry PREMIS_NAME "Enter the name of the database:" "60")
+        </hbox>
+        <hbox>
+          <checkbox>
+            <label>Auto update MySQL .plist?</label>
+            <variable>PREMIS_PLIST</variable>
+          </checkbox>
+          <checkbox>
+            <label>Generate video fingerprint?</label>
+            <variable>PERCEPTUAL_HASH</variable>
+          </checkbox>
+        </hbox>
+      </frame>
+      <frame Email Settings>
+        <hbox homogeneous="true">
+          $(funcEntry EMAIL_FROM "Please enter the email account that notifications will be sent from:" "80")
+          $(funcEntry SQL_ERROR_EMAIL_TO "Please the email account for the delivery of SQL Error Reports:" "80")
+        </hbox>
+        <hbox homogeneous="true">
+          $(funcEntry MAKEBROADCAST_DELIVERY_EMAIL_TO "Please enter the email account (or multiple accounts separated by a comma) makebroadcast notifications will be delivered to:" "80")
+          $(funcEntry MAKEYOUTUBE_DELIVERY_EMAIL_TO "Please enter the email account (or multiple accounts separated by a comma) makeyoutube notifications will be delivered to:" "80")
+        </hbox>
+      </frame>
+      <hbox>
+        <button ok></button>
+        <button cancel></button>
+      </hbox>
+   </vbox>
+  </window>
+MAIN_FORM
+)
+
+if EXIT="OK" ;
+  then GTK_OUTPUT="$(gtkdialog --program=MAIN_DIALOG | sort)"
+  echo "${GTK_OUTPUT}" | sed 's:VOLADJUSTY="true":VOLADJUST="Y":g'';''s:VOLADJUSTN="false"::g'';''s:VOLADJUSTN="true":VOLADJUST="N":g'';''s:VOLADJUSTY="false"::g'';''s:PREMIS_DBY="true":PREMIS_DB="Y":g'';''s:PREMIS_DBN="false"::g'';''s:PREMIS_DBN="true":PREMIS_DB="N":g'';''s:PREMIS_DBY="false"::g'';''s:EXIT="OK"::g'';' | sort > "${MM_CONFIG_FILE}"
```

Comment:
I think this could be

echo "${GTK_OUTPUT}" | sed 's:VOLADJUSTY="true":VOLADJUST="Y":g;s:VOLADJUST[YN]="false"::g;s:VOLADJUSTN="true":VOLADJUST="N":g;s:PREMIS_DBY="true":PREMIS_DB="Y":g;s:PREMIS_DB[YN]="false"::g;s:PREMIS_DBN="true":PREMIS_DB="N":g;s:EXIT="OK"::g' | sort > "${MM_CONFIG_FILE}"